ui/imports/shared/views/HistoryView.qml

```diff
@@ -136,29 +136,47 @@ ColumnLayout {
         ScrollBar.vertical: StatusScrollBar {}
 
         section.property: "date"
-        section.delegate: Item {
+        section.delegate: ColumnLayout {
             id: sectionDelegate
 
             readonly property bool isFirstSection: ListView.view.firstSection === section
 
             width: ListView.view.width
-            height: isFirstSection || section.length > 1 ? 40 : 0 // 1 because we don't use empty for loading state
+            height: isFirstSection ? 38 : (section.length > 1 ? 58 : 0) // 1 because we don't use empty for loading state
             visible: height > 0 // display always first section. Other sections when more items are being fetched must not be visible
+            spacing: 0
 
             required property string section
 
+            Item {
+                Layout.fillWidth: true
+                Layout.preferredHeight: 20
+                visible: !sectionDelegate.isFirstSection
+            }
+
+            Rectangle {
+                Layout.fillWidth: true
+                Layout.preferredHeight: 1
```

Comment:
Instead of having an `Item` above, is it possible to have here `Layout.topMargin: sectionDelegate.isFirstSection? 20 : 0`?